modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DocumentLibrary/DocumentLibrary.es.js

```diff
@@ -280,6 +364,56 @@ const Main = ({
 		itemSelectorDialog.open();
 	};
 
+	const configureErrorMessage = (message) => {
+		const enable = message ? true : false;
+
+		setErrorMessage(message);
+		setDisplayErrors(enable);
+		setValid(!enable);
+	};
+
+	const disableSubmitButton = (disable = true) => {
+		document.getElementById('ddm-form-submit').disabled = disable;
+	};
+
+	const handleUploadSelectButtonClicked = (event) => {
+		const data = {
+			[`${portletNamespace}file`]: event.target.files[0],
+		};
+
+		axios
+			.post(uploadURL, convertToFormData(data), {
+				onUploadProgress: (event) => {
+					const progress = Math.round(
+						(event.loaded * 100) / event.total
+					);
+
+					setCurrentValue(null);
+
+					setProgress(progress);
+
+					disableSubmitButton();
+				},
+			})
+			.then((response) => {
+				const {error, file} = response.data;
+
+				disableSubmitButton(false);
+
+				if (error) {
+					configureErrorMessage(error.message);
+					setCurrentValue(null);
+					onChange(event, '{}');
+				}
+				else {
+					configureErrorMessage('');
+					setCurrentValue(JSON.stringify(file));
+					onChange(event, JSON.stringify(file));
+				}
+				setProgress(0);
```

Comment:
@alinedoleron, please add line breakers among these lines to separate the methods invocations. Please check how you did it on line 391 until line 395. Let's make it consistent. Keep it in mind. 🙂